meta_aggregation_api/rest_api/routes/gas.py

```diff
@@ -1,25 +1,22 @@
-from fastapi import Path, Depends
+from aiocache import cached
+from fastapi import Path
 from fastapi.routing import APIRouter
-from fastapi.security import HTTPBearer
-from fastapi_jwt_auth import AuthJWT
 
 from meta_aggregation_api.models.gas_models import GasResponse
 from meta_aggregation_api.services.gas_service import get_gas_prices
 
+GAS_CACHE_TTL = 5
```

Comment:
nit: It would be good to specify what 5 means. Whether it is in seconds or minutes.
```suggestion
GAS_CACHE_TTL_SEC = 5
```